reviser/reviser.go

```diff
@@ -238,6 +238,19 @@ func generateFile(fset *token.FileSet, f *ast.File) ([]byte, error) {
 	return buffer.Bytes(), nil
 }
 
+func isSingleCgoImport(dd *ast.GenDecl) bool {
+	if dd.Tok != token.IMPORT {
+		return false
+	}
+	if len(dd.Specs) != 1 {
+		return false
+	}
+	if dd.Specs[0].(*ast.ImportSpec).Path.Value != `"C"` {
```

Comment:
return dd.Specs[0].(*ast.ImportSpec).Path.Value == `"C"`

Reply:
Changed it, thanks.